### ISSUE TITLE
Add a seed script in any new language

### ISSUE THREAD

**Issue by mkasberg** [mkasberg/script-seed#49]
Open a pull request to add a sample script for any scripting language we don't already have. For a language to be included, it needs to be **executable as a script (i.e. with a `#!` shebang) on Ubuntu Linux** (after installing required dependencies), and it needs to be able to parse basic command line arguments (like `-h`). See the existing scripts for examples.

Complete instructions for adding a new language are available in [**CONTRIBUTING**](https://github.com/mkasberg/script-seed/blob/master/CONTRIBUTING.md#how-to-add-a-language). Feel free to ask if you need help getting the tests working. Also, feel free to open an issue proposing the new language you want to add, and we can work through details before you start if you'd like.

If you want to write a language that we already have, consider improving the existing script instead. (We don't want duplicates.) I'm happy to merge substantive improvements (even if they're small), but I'm not interested in useless text changes (like renaming `plant` to `tree`).

As a rough example, see #26.

**Comment by paymin** [comment]
How about Golang? I would like to create it

**Comment by mkasberg** [comment]
Yes, it looks like a Golang script should be possible with the help of the [gorun](https://wiki.ubuntu.com/gorun#Ubuntu_packages) utility to run the go file as a script. I'd be happy to merge a PR that adds a sample Golang script.
